scripts/setup.py

```diff
@@ -43,7 +43,13 @@ def create_makefile_template_defines(
     with open(makefile, "w") as fileobj:
         print(SRCDIR_TEXT.format(os.path.relpath(source_root, proof_root)),
               file=fileobj)
-        print(LITANI_TEXT.format(os.path.relpath(litani, proof_root)),
+        if is_litani_installed:
+            litani_path = '"litani"'
+        else:
+            litani_path = util.read_litani_path()
+            relpath = os.path.relpath(litani_path, proof_root)
+            litani_path = f"$(abspath $(PROOF_ROOT)/{relpath})"
+        print(LITANI_TEXT.format(litani_path),
```

Comment:
I would remove the `is_litani_installed` parameter from the function and instead create a utils function called `get_litani_path()` which would do something like:

```python
def get_litani_path():
    if util.is_litani_installed():
        return pathlib.Path("litani")
    return util.read_litani_path()
```
that way the code here would be:
```python
litani_path = util.get_litani_path().relative_to(proof_root)
```
Also do we want to use the `os` or the `pathlib` libraries for paths? I'm more familiar with `pathlib` but it seems as though we're using `os` in the other script so you'll probably need to convert .

Reply:
Actually nevermind I don't think that would work how I thought it would

Reply:
The `is_litani_installed` parameter has been removed.

Reply:
> Also do we want to use the `os` or the `pathlib` libraries for paths?

I'm open to making that change. However, it should not be part of this PR as it points to a need to conduct a broader minor refactoring.